framework/wazuh/tests/test_configuration.py

```diff
@@ -78,6 +80,12 @@ def test_read_option():
     with open(os.path.join(parent_directory, tmp_path, 'configuration/default/options1.conf')) as f:
         data = fromstring(f.read())
         assert configuration._read_option('labels', data)[0] == 'label'
+        assert configuration._read_option('test', data) == ('label', {'name': 'first', 'item': 'test'})
+
+    with open(os.path.join(parent_directory, tmp_path, 'configuration/default/options2.conf')) as f:
```

Comment:
```suggestion
    with open(os.path.join(parent_directory, tmp_path, 'configuration/default/synchronization.conf')) as f:
```
I suggest renaming this file to understand what it is simply reading its name.